lib/appium_lib/device/device.rb

```diff
@@ -15,6 +15,7 @@ module Device
         device_locked?:       'session/:session_id/appium/device/is_locked'
       },
       get:  {
+        get_device_time:        'session/:sessionId/appium/device/system_time',
```

Comment:
Just wondering since all the other endpoints are `session/:session_id` but this one is `session/:sessionId`. Is this correct?